core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -245,18 +249,20 @@ object SparkSubmit {
     // from each other if only one is specified, or exit early if they are at odds.
     if (clusterManager == YARN) {
       if (args.master == "yarn-standalone") {
-        printWarning("\"yarn-standalone\" is deprecated. Use \"yarn-cluster\" instead.")
+        printWarning("\"yarn-standalone\" is deprecated. Use \"yarn\" " +
```

Comment:
I'm planning to remove the support of "yarn-standalone". Like what @srowen said, I don't there's many user still use "yarn-standalone" for now.